src/Analysis/Codelyzer.Analysis/AnalyzerCLI.cs

```diff
@@ -42,9 +42,11 @@ public class Options
         [Option('x', "msbuild-location", Required =false, HelpText ="MSBuild exe to be used for building the solution")]
         public string MSBuildLocation { get; set; }
 
+        [Option('y', "build-only", Required = false, HelpText = "Runs build without any analysis")]
+        public string BuildOnly { get; set; }
+
```

Comment:
can't we use build-arguments ?

Reply:
We can, but build arguments are meant to be passed directly to the build command. If we use them, we'll have to start creating specail cases for arguments we are using.